site/next.config.js

```diff
@@ -33,6 +33,11 @@ module.exports = withFonts(
           destination: "/contact/submitted",
           permanent: true,
         },
+        {
+          source: "/dashboard/create-schema",
+          destination: "/dashboard?create-schema=true",
+          permanent: true,
+        },
```

Comment:
@dw, Create Schema page can be accessed by going to  `/dashboard?create-schema=true` or better still `/dashboard/create-schema`. If the user is not authenticated, they are taken to the login screen and redirected back to the page on successful login